ibm/service/power/ibm_pi_constants.go

```diff
@@ -4,31 +4,38 @@ import "time"
 
 const (
 	// Arguments
-	Arg_CloudConnectionName                 = "pi_cloud_connection_name"
-	Arg_CloudInstanceID                     = "pi_cloud_instance_id"
-	Arg_ImageName                           = "pi_image_name"
-	Arg_InstanceName                        = "pi_instance_name"
-	Arg_Key                                 = "pi_ssh_key"
-	Arg_KeyName                             = "pi_key_name"
-	Arg_NetworkName                         = "pi_network_name"
-	Arg_PlacementGroupName                  = "pi_placement_group_name"
-	Arg_SAP                                 = "sap"
-	Arg_SAPProfileID                        = "pi_sap_profile_id"
-	Arg_SPPPlacementGroupID                 = "pi_spp_placement_group_id"
-	Arg_SPPPlacementGroupName               = "pi_spp_placement_group_name"
-	Arg_SPPPlacementGroupPolicy             = "pi_spp_placement_group_policy"
-	Arg_SharedProcessorPoolHostGroup        = "pi_shared_processor_pool_host_group"
-	Arg_SharedProcessorPoolID               = "pi_shared_processor_pool_id"
-	Arg_SharedProcessorPoolName             = "pi_shared_processor_pool_name"
-	Arg_SharedProcessorPoolPlacementGroupID = "pi_shared_processor_pool_placement_group_id"
-	Arg_SharedProcessorPoolReservedCores    = "pi_shared_processor_pool_reserved_cores"
-	Arg_SnapshotID                          = "pi_snapshot_id"
-	Arg_StoragePool                         = "pi_storage_pool"
-	Arg_StorageType                         = "pi_storage_type"
-	Arg_VTL                                 = "vtl"
-	Arg_VolumeGroupID                       = "pi_volume_group_id"
-	Arg_VolumeID                            = "pi_volume_id"
-	Arg_VolumeOnboardingID                  = "pi_volume_onboarding_id"
+	Arg_CloudConnectionName                  = "pi_cloud_connection_name"
+	Arg_CloudInstanceID                      = "pi_cloud_instance_id"
+	Arg_ImageName                            = "pi_image_name"
+	Arg_InstanceCaptureCloudStorageAccessKey = "pi_capture_cloud_storage_access_key"
+	Arg_InstanceCaptureCloudStorageImagePath = "pi_capture_storage_image_path"
+	Arg_InstanceCaptureCloudStorageSecretKey = "pi_capture_cloud_storage_secret_key"
+	Arg_InstanceCaptureCloudStorageRegion    = "pi_capture_cloud_storage_region"
+	Arg_InstanceCaptureDestination           = "pi_capture_destination"
+	Arg_InstanceCaptureName                  = "pi_capture_name"
+	Arg_InstanceCaptureVolumeIds             = "pi_capture_volume_ids"
+	Arg_InstanceName                         = "pi_instance_name"
```

Comment:
All those should be name just like `Arg_InstanceName`
Prefix+Camel case
this: `Arg_InstanceCaptureCloudStorageAccessKey = "pi_capture_cloud_storage_access_key"` should be Arg_CaptureCloudStorageAccessKey = "pi_capture_cloud_storage_access_key"